addons/website/static/src/js/menu/edit.js

```diff
@@ -355,7 +355,8 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         // this badly relies on the contenteditable="true" attribute being on
         // those images but it is rightfully lost after the first save.
         // grep: COMPANY_TEAM_CONTENTEDITABLE
-        let $extraEditableZones = $editableSavableZones.find('.s_company_team .o_not_editable img');
+        const teamSnippetMediaCol = ".s_company_team .o_not_editable";
+        let $extraEditableZones = $editableSavableZones.find(`${teamSnippetMediaCol} img, ${teamSnippetMediaCol} span.fa, ${teamSnippetMediaCol} .media_iframe_video`);
```

Comment:
@qsm-odoo Note that once task [3226172](https://www.odoo.com/web#menu_id=4720&cids=1&action=333&active_id=1695&model=project.task&view_type=form&id=3226172) is merged, we will be able to use `o_editable_media` here to reduce the code and make it more generic. This will also prevent the selector from targeting elements that should not be targeted, such as elements with the `o_not_editable` class and media inside it, within a potential "inner-content" snippet dropped into the Team snippet.